parseShort.py

```diff
@@ -0,0 +1,53 @@
+#!/bin/python
+import re;
+import os;
+
+fName = "diag_manager.F90"
+
+def getModules(fileName):
+	MODS = [] 
+	
+	fileContents = open(fileName).read()
+	
+	#The regex creates a pattern to find matches in the file
+	#Checks for USE and possible & on the same or next line, until module name is found signified by the '?'
+	#Ignores cases and tries to match on all lines
```

Comment:
Use doxygen comments, preferably the ## style suggested [here](http://www.doxygen.nl/manual/docblocks.html#pythonblocks)